src/components/program/ProgramContentMenu.tsx

```diff
@@ -232,13 +239,11 @@ const SortBySectionItem: React.FC<{
 
   return (
     <StyledItem
-      className={`${progressStatus} ${isActive ? 'active' : ''}`}
+      className={`${progressStatus} ${isActive && 'active'}`}
```

Comment:
it will be `false` in string type when `isActive` is false, `<StyledItem />` will get a dummy class